It is going to be quite hard to launch the cadvisor e2e tests.

```
maha4472@knative-e2e:~/dims-kubernetes$ make test-e2e-node REMOTE=true REMOTE_MODE=gce TEST_SUITE=cadvisor IMAGE_CONFIG_FILE=~/k8s-test-infra/jobs/e2e_node/containerd/image-cadvisor.yaml FOCUS="\[NodeConformance\]" 
+++ [0314 10:26:27] WARNING: linux/arm will no longer be built/shipped by default, please build it explicitly if needed.
+++ [0314 10:26:27]          support for linux/arm will be removed in a subsequent release.
go version go1.20.2 linux/amd64
+++ [0314 10:26:28] Building go targets for linux/amd64
    github.com/onsi/ginkgo/v2/ginkgo (non-static)
+++ [0314 10:26:29] WARNING: linux/arm will no longer be built/shipped by default, please build it explicitly if needed.
+++ [0314 10:26:29]          support for linux/arm will be removed in a subsequent release.
go version go1.20.2 linux/amd64
Creating artifacts directory at /tmp/_artifacts/230314T102629
Test artifacts will be written to /tmp/_artifacts/230314T102629
Running tests remotely using
Project: coen-mahamed-ali
Image Project: cos-cloud
Compute/Zone: europe-west4-b
Images: 
Hosts: 
Ginkgo Flags: -timeout=24h -nodes=8  -focus="\[NodeConformance\]"  -skip="\[Flaky\]|\[Slow\]|\[Serial\]"  -untilItFails=false 
Instance Metadata: 
Image Config File: /home/maha4472/k8s-test-infra/jobs/e2e_node/containerd/image-cadvisor.yaml
Kubelet Config File: test/e2e_node/jenkins/default-kubelet-config.yaml
Will use ginkgo flags as: -timeout=24h -nodes=8  -focus="\[NodeConformance\]"  -skip="\[Flaky\]|\[Slow\]|\[Serial\]"  -untilItFails=false  --no-color -vI0314 10:26:36.056067  671358 gce_runner.go:243] found images [{creationTime:{wall:587000000 ext:63813822869 loc:0xc0003102a0} name:cos-89-16108-798-17} {creationTime:{wall:865000000 ext:63812001035 loc:0xc0003102a0} name:cos-89-16108-798-10} {creationTime:{wall:954000000 ext:63810961936 loc:0xc0003102a0} name:cos-89-16108-798-7} {creationTime:{wall:792000000 ext:63810100201 loc:0xc0003102a0} name:cos-89-16108-798-3} {creationTime:{wall:649000000 ext:63809597425 loc:0xc0003102a0} name:cos-89-16108-798-1} {creationTime:{wall:932000000 ext:63808897061 loc:0xc0003102a0} name:cos-89-16108-766-29} {creationTime:{wall:762000000 ext:63806474750 loc:0xc0003102a0} name:cos-89-16108-766-19} {creationTime:{wall:283000000 ext:63805873591 loc:0xc0003102a0} name:cos-89-16108-766-15} {creationTime:{wall:704000000 ext:63803715594 loc:0xc0003102a0} name:cos-89-16108-766-13} {creationTime:{wall:231000000 ext:63803458733 loc:0xc0003102a0} name:cos-89-16108-766-9} {creationTime:{wall:955000000 ext:63802838617 loc:0xc000310310} name:cos-89-16108-766-5} {creationTime:{wall:259000000 ext:63802243584 loc:0xc000310310} name:cos-89-16108-766-3} {creationTime:{wall:58000000 ext:63801630317 loc:0xc000310310} name:cos-89-16108-766-1} {creationTime:{wall:919000000 ext:63799816926 loc:0xc000310310} name:cos-89-16108-717-35} {creationTime:{wall:18000000 ext:63799224874 loc:0xc000310310} name:cos-89-16108-717-34} {creationTime:{wall:623000000 ext:63798608704 loc:0xc000310310} name:cos-89-16108-717-30} {creationTime:{wall:605000000 ext:63798096374 loc:0xc000310310} name:cos-89-16108-717-20} {creationTime:{wall:740000000 ext:63797397279 loc:0xc000310310} name:cos-89-16108-717-17} {creationTime:{wall:104000000 ext:63796796724 loc:0xc000310310} name:cos-89-16108-717-14} {creationTime:{wall:616000000 ext:63796194179 loc:0xc000310310} name:cos-89-16108-717-11} {creationTime:{wall:147000000 ext:63795591359 loc:0xc000310310} name:cos-89-16108-717-9} {creationTime:{wall:785000000 ext:63794994361 loc:0xc000310310} name:cos-89-16108-717-6} {creationTime:{wall:258000000 ext:63794380918 loc:0xc000310310} name:cos-89-16108-717-4} {creationTime:{wall:402000000 ext:63793767754 loc:0xc000310310} name:cos-89-16108-717-3} {creationTime:{wall:938000000 ext:63793345869 loc:0xc000310310} name:cos-89-16108-717-1} {creationTime:{wall:78000000 ext:63793170946 loc:0xc000310310} name:cos-89-16108-659-29} {creationTime:{wall:138000000 ext:63792739333 loc:0xc000310310} name:cos-89-16108-659-28} {creationTime:{wall:747000000 ext:63791959107 loc:0xc000310310} name:cos-89-16108-659-24} {creationTime:{wall:277000000 ext:63789890856 loc:0xc000310310} name:cos-89-16108-659-19} {creationTime:{wall:524000000 ext:63789111933 loc:0xc000310310} name:cos-89-16108-659-15} {creationTime:{wall:230000000 ext:63788938335 loc:0xc000310310} name:cos-89-16108-659-14} {creationTime:{wall:408000000 ext:63786522364 loc:0xc000310310} name:cos-89-16108-659-8} {creationTime:{wall:413000000 ext:63785903875 loc:0xc000310310} name:cos-89-16108-659-6} {creationTime:{wall:663000000 ext:63785402958 loc:0xc000310310} name:cos-89-16108-659-1} {creationTime:{wall:463000000 ext:63784798596 loc:0xc000310310} name:cos-89-16108-604-31} {creationTime:{wall:530000000 ext:63783826053 loc:0xc000310310} name:cos-89-16108-604-28} {creationTime:{wall:931000000 ext:63783484752 loc:0xc000310310} name:cos-89-16108-604-22} {creationTime:{wall:742000000 ext:63782282659 loc:0xc0003102a0} name:cos-89-16108-604-19} {creationTime:{wall:611000000 ext:63781677746 loc:0xc0003102a0} name:cos-89-16108-604-17} {creationTime:{wall:488000000 ext:63780473831 loc:0xc0003102a0} name:cos-89-16108-604-11} {creationTime:{wall:926000000 ext:63779857886 loc:0xc0003102a0} name:cos-89-16108-604-5} {creationTime:{wall:124000000 ext:63778234952 loc:0xc0003102a0} name:cos-89-16108-604-3} {creationTime:{wall:894000000 ext:63778145410 loc:0xc0003102a0} name:cos-89-16108-604-1} {creationTime:{wall:878000000 ext:63777716785 loc:0xc0003102a0} name:cos-89-16108-534-43} {creationTime:{wall:354000000 ext:63777536789 loc:0xc0003102a0} name:cos-89-16108-534-41} {creationTime:{wall:339000000 ext:63775025706 loc:0xc0003102a0} name:cos-89-16108-534-34} {creationTime:{wall:391000000 ext:63773987449 loc:0xc0003102a0} name:cos-89-16108-534-27} {creationTime:{wall:713000000 ext:63772609558 loc:0xc0003102a0} name:cos-89-16108-534-22} {creationTime:{wall:376000000 ext:63771579720 loc:0xc000310310} name:cos-89-16108-534-18} {creationTime:{wall:270000000 ext:63770183719 loc:0xc000310310} name:cos-89-16108-534-17} {creationTime:{wall:446000000 ext:63769619562 loc:0xc000310310} name:cos-89-16108-534-13} {creationTime:{wall:495000000 ext:63768969685 loc:0xc000310310} name:cos-89-16108-534-9} {creationTime:{wall:596000000 ext:63768367192 loc:0xc000310310} name:cos-89-16108-534-8} {creationTime:{wall:86000000 ext:63767759045 loc:0xc000310310} name:cos-89-16108-534-2} {creationTime:{wall:754000000 ext:63767161604 loc:0xc000310310} name:cos-89-16108-470-25} {creationTime:{wall:821000000 ext:63765362836 loc:0xc000310310} name:cos-89-16108-470-16} {creationTime:{wall:354000000 ext:63762932972 loc:0xc000310310} name:cos-89-16108-470-11} {creationTime:{wall:861000000 ext:63760161314 loc:0xc000310310} name:cos-89-16108-470-1} {creationTime:{wall:569000000 ext:63759904349 loc:0xc000310310} name:cos-89-16108-403-51} {creationTime:{wall:244000000 ext:63759307586 loc:0xc000310310} name:cos-89-16108-403-47} {creationTime:{wall:436000000 ext:63758783016 loc:0xc000310310} name:cos-89-16108-403-46} {creationTime:{wall:366000000 ext:63758701050 loc:0xc000310310} name:cos-89-16108-403-42} {creationTime:{wall:150000000 ext:63755667424 loc:0xc000310310} name:cos-89-16108-403-26} {creationTime:{wall:577000000 ext:63754716413 loc:0xc000310310} name:cos-89-16108-403-22} {creationTime:{wall:573000000 ext:63753426856 loc:0xc000310310} name:cos-89-16108-403-15}] based on regex "" and family "cos-89-lts" in project "cos-cloud"
I0314 10:26:36.056201  671358 gce_runner.go:354] parsing instance metadata: "user-data</workspace/test-infra/jobs/e2e_node/containerd/init.yaml,cni-template</workspace/test-infra/jobs/e2e_node/containerd/cni.template,containerd-config</workspace/test-infra/jobs/e2e_node/containerd/config.toml"
I0314 10:26:36.056277  671358 gce_runner.go:356] parsed instance metadata: map[cni-template:{
  "name": "k8s-pod-network",
  "cniVersion": "0.3.1",
  "plugins": [
    {
      "type": "ptp",
      "mtu": 1460,
      "ipam": {
        "type": "host-local",
        "subnet": "{{.PodCIDR}}",
        "routes": [
          {
            "dst": "0.0.0.0/0"
          }
        ]
      }
    },
    {
      "type": "portmap",
      "capabilities": {
        "portMappings": true
      }
    }
  ]
}
 containerd-config:version = 2
required_plugins = ["io.containerd.grpc.v1.cri"]
# Kubernetes doesn't use containerd restart manager.
disabled_plugins = ["io.containerd.internal.v1.restart"]
oom_score = -999

[debug]
  level = "debug"

[plugins."io.containerd.grpc.v1.cri"]
  stream_server_address = "127.0.0.1"
  max_container_log_line_size = 262144
[plugins."io.containerd.grpc.v1.cri".cni]
  bin_dir = "/home/containerd/"
  conf_dir = "/etc/cni/net.d"
  conf_template = "/home/containerd/cni.template"
[plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
  endpoint = ["https://mirror.gcr.io","https://registry-1.docker.io"]

[plugins."io.containerd.grpc.v1.cri".containerd]
  default_runtime_name = "runc"
[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
  runtime_type = "io.containerd.runc.v2"

# Runtime handler used for runtime class test.
[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.test-handler]
  runtime_type = "io.containerd.runc.v2"

# Enable registry.k8s.io as the primary mirror for k8s.gcr.io
# See: https://github.com/kubernetes/k8s.io/issues/3411
[plugins."io.containerd.grpc.v1.cri".registry.mirrors."k8s.gcr.io"]
  endpoint = ["https://registry.k8s.io", "https://k8s.gcr.io",]
 user-data:#cloud-config

runcmd:
  - echo "Test run from /tmp folder, remounting it"
  - mount /tmp /tmp -o remount,exec,suid

  - echo "This will configure built-in containerd for k8s tests. Containerd version is:"
  - ctr version # current version of containerd

  - echo "Download and install CNI configuration to /home/containerd"
  - mkdir -p /home/containerd
  - mount --bind /home/containerd /home/containerd
  - mount -o remount,exec /home/containerd
  - 'curl --fail --retry 5 --retry-delay 3 --silent --show-error -H "X-Google-Metadata-Request: True" -o /home/containerd/cni.template http://metadata.google.internal/computeMetadata/v1/instance/attributes/cni-template'

  - echo "Download and install CNI to /home/containerd"
  - 'curl -sSL --fail --retry 5 --retry-delay 3 --silent --show-error -o /home/containerd/cni.tgz https://storage.googleapis.com/k8s-artifacts-cni/release/v1.0.1/cni-plugins-linux-amd64-v1.0.1.tgz'
  - tar xzf /home/containerd/cni.tgz -C /home/containerd --overwrite

  - echo "Set containerd configuration"
  - mkdir -p /etc/containerd
  - 'curl --fail --retry 5 --retry-delay 3 --silent --show-error -H "X-Google-Metadata-Request: True" -o /etc/containerd/config.toml http://metadata.google.internal/computeMetadata/v1/instance/attributes/containerd-config'

  - echo "Restarting containerd"
  - systemctl restart containerd

  - echo "Configuration complete"
]
I0314 10:26:39.108677  671358 gce_runner.go:243] found images [{creationTime:{wall:959000000 ext:63814351799 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20230313} {creationTime:{wall:351000000 ext:63813830533 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230307} {creationTime:{wall:62000000 ext:63813486505 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230303} {creationTime:{wall:805000000 ext:63813140321 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230227} {creationTime:{wall:372000000 ext:63812771512 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230222} {creationTime:{wall:498000000 ext:63812275190 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230217} {creationTime:{wall:587000000 ext:63812189535 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230216} {creationTime:{wall:955000000 ext:63812016389 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230214} {creationTime:{wall:581000000 ext:63811930358 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230213} {creationTime:{wall:441000000 ext:63811450492 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230207} {creationTime:{wall:426000000 ext:63811325024 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230206} {creationTime:{wall:360000000 ext:63810978564 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230202} {creationTime:{wall:977000000 ext:63810864607 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230130} {creationTime:{wall:739000000 ext:63809756691 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230119} {creationTime:{wall:590000000 ext:63809595574 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230117} {creationTime:{wall:515000000 ext:63809509444 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230116} {creationTime:{wall:748000000 ext:63809163620 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230112a} {creationTime:{wall:955000000 ext:63809124189 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230112} {creationTime:{wall:558000000 ext:63808991569 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230110} {creationTime:{wall:57000000 ext:63808731710 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230107} {creationTime:{wall:94000000 ext:63808537190 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230105} {creationTime:{wall:198000000 ext:63808388618 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20230103} {creationTime:{wall:834000000 ext:63806744124 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20221215} {creationTime:{wall:629000000 ext:63806485325 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20221212} {creationTime:{wall:875000000 ext:63806138805 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20221208} {creationTime:{wall:458000000 ext:63805966699 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20221206} {creationTime:{wall:621000000 ext:63805926004 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20221205} {creationTime:{wall:774000000 ext:63805624577 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20221202} {creationTime:{wall:483000000 ext:63805509854 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20221201} {creationTime:{wall:732000000 ext:63805280306 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20221128} {creationTime:{wall:43000000 ext:63804843176 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20221123} {creationTime:{wall:996000000 ext:63804670647 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20221121} {creationTime:{wall:89000000 ext:63804419234 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20221118} {creationTime:{wall:359000000 ext:63804240725 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20221116} {creationTime:{wall:148000000 ext:63804153264 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20221115} {creationTime:{wall:111000000 ext:63803635469 loc:0xc0003102a0} name:ubuntu-gke-2204-1-24-v20221109} {creationTime:{wall:753000000 ext:63803201362 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20221104} {creationTime:{wall:390000000 ext:63803094766 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20221103} {creationTime:{wall:258000000 ext:63802922349 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20221101} {creationTime:{wall:939000000 ext:63802489655 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20221027} {creationTime:{wall:441000000 ext:63802410365 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20221026} {creationTime:{wall:186000000 ext:63802339621 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20221025} {creationTime:{wall:415000000 ext:63801737200 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20221018} {creationTime:{wall:531000000 ext:63801386971 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20221014} {creationTime:{wall:900000000 ext:63801057270 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20221010} {creationTime:{wall:115000000 ext:63800609674 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20221005} {creationTime:{wall:995000000 ext:63800439615 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20221003} {creationTime:{wall:291000000 ext:63799980529 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220928} {creationTime:{wall:759000000 ext:63798886047 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220915} {creationTime:{wall:760000000 ext:63798279342 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220908} {creationTime:{wall:204000000 ext:63798244707 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220906} {creationTime:{wall:720000000 ext:63797848726 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220903} {creationTime:{wall:950000000 ext:63797418207 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220829} {creationTime:{wall:171000000 ext:63797157474 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220826} {creationTime:{wall:376000000 ext:63796845408 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220822} {creationTime:{wall:410000000 ext:63796421222 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220817} {creationTime:{wall:222000000 ext:63795740458 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220810} {creationTime:{wall:592000000 ext:63795551960 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220807} {creationTime:{wall:472000000 ext:63795082961 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220802} {creationTime:{wall:80000000 ext:63794825225 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220730} {creationTime:{wall:12000000 ext:63794716430 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220729} {creationTime:{wall:310000000 ext:63794524567 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220727} {creationTime:{wall:520000000 ext:63793984191 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220721} {creationTime:{wall:777000000 ext:63793897628 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220720} {creationTime:{wall:866000000 ext:63793613950 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220716} {creationTime:{wall:190000000 ext:63793465118 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220715} {creationTime:{wall:311000000 ext:63793378437 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220714} {creationTime:{wall:736000000 ext:63792889496 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220707} {creationTime:{wall:458000000 ext:63792687904 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220706} {creationTime:{wall:79000000 ext:63792664321 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220705} {creationTime:{wall:378000000 ext:63792577470 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220704} {creationTime:{wall:493000000 ext:63792169671 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220630} {creationTime:{wall:129000000 ext:63791685565 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220624} {creationTime:{wall:397000000 ext:63791668745 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220624} {creationTime:{wall:807000000 ext:63791585574 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220623} {creationTime:{wall:264000000 ext:63791495780 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220622} {creationTime:{wall:521000000 ext:63791329955 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220620} {creationTime:{wall:706000000 ext:63790933412 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220615} {creationTime:{wall:242000000 ext:63790760796 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220613a} {creationTime:{wall:495000000 ext:63790613880 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220612} {creationTime:{wall:795000000 ext:63790354816 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220609} {creationTime:{wall:377000000 ext:63790181905 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220607} {creationTime:{wall:503000000 ext:63790155868 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220606} {creationTime:{wall:509000000 ext:63789753790 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220601} {creationTime:{wall:260000000 ext:63789599966 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220531} {creationTime:{wall:682000000 ext:63789535493 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220530a} {creationTime:{wall:710000000 ext:63789233908 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220527} {creationTime:{wall:967000000 ext:63789212565 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220526} {creationTime:{wall:902000000 ext:63789175663 loc:0xc000310310} name:ubuntu-gke-2004-1-24-v20220525} {creationTime:{wall:273000000 ext:63788341811 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220516} {creationTime:{wall:353000000 ext:63788082649 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220513} {creationTime:{wall:855000000 ext:63787945883 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220511} {creationTime:{wall:826000000 ext:63787824704 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220510} {creationTime:{wall:862000000 ext:63787430994 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220506} {creationTime:{wall:128000000 ext:63786150003 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220420} {creationTime:{wall:334000000 ext:63785922688 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220418} {creationTime:{wall:573000000 ext:63785749473 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220416} {creationTime:{wall:670000000 ext:63785663128 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220415} {creationTime:{wall:846000000 ext:63785577168 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220414} {creationTime:{wall:37000000 ext:63785475322 loc:0xc000310310} name:ubuntu-gke-2204-1-24-v20220412}] based on regex "" and family "pipeline-1-24" in project "ubuntu-os-gke-cloud"
I0314 10:26:39.110262  671358 gce_runner.go:354] parsing instance metadata: "user-data</workspace/test-infra/jobs/e2e_node/containerd/init.yaml,cni-template</workspace/test-infra/jobs/e2e_node/containerd/cni.template,containerd-config</workspace/test-infra/jobs/e2e_node/containerd/config.toml"
I0314 10:26:39.110448  671358 gce_runner.go:356] parsed instance metadata: map[cni-template:{
  "name": "k8s-pod-network",
  "cniVersion": "0.3.1",
  "plugins": [
    {
      "type": "ptp",
      "mtu": 1460,
      "ipam": {
        "type": "host-local",
        "subnet": "{{.PodCIDR}}",
        "routes": [
          {
            "dst": "0.0.0.0/0"
          }
        ]
      }
    },
    {
      "type": "portmap",
      "capabilities": {
        "portMappings": true
      }
    }
  ]
}
 containerd-config:version = 2
required_plugins = ["io.containerd.grpc.v1.cri"]
# Kubernetes doesn't use containerd restart manager.
disabled_plugins = ["io.containerd.internal.v1.restart"]
oom_score = -999

[debug]
  level = "debug"

[plugins."io.containerd.grpc.v1.cri"]
  stream_server_address = "127.0.0.1"
  max_container_log_line_size = 262144
[plugins."io.containerd.grpc.v1.cri".cni]
  bin_dir = "/home/containerd/"
  conf_dir = "/etc/cni/net.d"
  conf_template = "/home/containerd/cni.template"
[plugins."io.containerd.grpc.v1.cri".registry.mirrors."docker.io"]
  endpoint = ["https://mirror.gcr.io","https://registry-1.docker.io"]

[plugins."io.containerd.grpc.v1.cri".containerd]
  default_runtime_name = "runc"
[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
  runtime_type = "io.containerd.runc.v2"

# Runtime handler used for runtime class test.
[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.test-handler]
  runtime_type = "io.containerd.runc.v2"

# Enable registry.k8s.io as the primary mirror for k8s.gcr.io
# See: https://github.com/kubernetes/k8s.io/issues/3411
[plugins."io.containerd.grpc.v1.cri".registry.mirrors."k8s.gcr.io"]
  endpoint = ["https://registry.k8s.io", "https://k8s.gcr.io",]
 user-data:#cloud-config

runcmd:
  - echo "Test run from /tmp folder, remounting it"
  - mount /tmp /tmp -o remount,exec,suid

  - echo "This will configure built-in containerd for k8s tests. Containerd version is:"
  - ctr version # current version of containerd

  - echo "Download and install CNI configuration to /home/containerd"
  - mkdir -p /home/containerd
  - mount --bind /home/containerd /home/containerd
  - mount -o remount,exec /home/containerd
  - 'curl --fail --retry 5 --retry-delay 3 --silent --show-error -H "X-Google-Metadata-Request: True" -o /home/containerd/cni.template http://metadata.google.internal/computeMetadata/v1/instance/attributes/cni-template'

  - echo "Download and install CNI to /home/containerd"
  - 'curl -sSL --fail --retry 5 --retry-delay 3 --silent --show-error -o /home/containerd/cni.tgz https://storage.googleapis.com/k8s-artifacts-cni/release/v1.0.1/cni-plugins-linux-amd64-v1.0.1.tgz'
  - tar xzf /home/containerd/cni.tgz -C /home/containerd --overwrite

  - echo "Set containerd configuration"
  - mkdir -p /etc/containerd
  - 'curl --fail --retry 5 --retry-delay 3 --silent --show-error -H "X-Google-Metadata-Request: True" -o /etc/containerd/config.toml http://metadata.google.internal/computeMetadata/v1/instance/attributes/containerd-config'

  - echo "Restarting containerd"
  - systemctl restart containerd

  - echo "Configuration complete"
]
I0314 10:26:39.111050  671358 remote.go:72] Building archive...
+ BUILDER=false
+ export GO_FLAGS=-race
+ GO_FLAGS=-race
+ export GORACE=halt_on_error=1
+ GORACE=halt_on_error=1
+++ dirname /home/maha4472/dims-kubernetes/_output/local/go/src/github.com/google/cadvisor/build/prow_e2e.sh
++ cd /home/maha4472/dims-kubernetes/_output/local/go/src/github.com/google/cadvisor/build
++ pwd -P
+ parent_path=/home/maha4472/dims-kubernetes/_output/local/go/src/github.com/google/cadvisor/build
+ cd /home/maha4472/dims-kubernetes/_output/local/go/src/github.com/google/cadvisor/build/..
+ FORCE=true
+ build/assets.sh
go: modules disabled by GO111MODULE=off; see 'go help modules'
2023/03/14 10:26:39 unable to create test archive: failed to setup test package "/tmp/node-e2e-archive1375218196": failed to run command /home/maha4472/dims-kubernetes/_output/local/go/src/github.com/google/cadvisor/build/prow_e2e.sh. error: exit status 1
exit status 1
make: *** [Makefile:274: test-e2e-node] Error 1
maha4472@knative-e2e:~/dims-kubernetes$ cat /home/maha4472/dims-kubernetes/_output/local/go/src/github.com/google/cadvisor/build/prow_e2e.sh
#!/usr/bin/env bash

# Copyright 2018 Google Inc. All rights reserved.
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#     http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

set -e
set -x

BUILDER=${BUILDER:-false} # Whether this is running a PR builder job.

export GO_FLAGS="-race"
export GORACE="halt_on_error=1"

# cd to cadvisor directory
parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
cd "$parent_path/.."

# Check whether assets need to be rebuilt.
FORCE=true build/assets.sh
if [[ ! -z "$(git diff --name-only -- cmd/internal/pages)" ]]; then
  echo "Found changes to UI assets:"
  git diff --name-only -- cmd/internal/pages
  echo "Run: 'make assets FORCE=true'"
  exit 1
fi

make all

# compile integration tests so they can be run without go installed
go test -c github.com/google/cadvisor/integration/tests/api
go test -c github.com/google/cadvisor/integration/tests/healthz
```

--- a/test/e2e_node/runner/remote/run_remote.go
+++ b/test/e2e_node/runner/remote/run_remote.go
@@ -22,13 +22,19 @@ package main
 
 import (
 	"flag"
+	"fmt"
 
 	"k8s.io/klog/v2"
 
 	"k8s.io/kubernetes/test/e2e_node/remote"
+	"k8s.io/kubernetes/test/e2e_node/system"
 )
 
 var testSuite = flag.String("test-suite", "default", "Test suite the runner initializes with. Currently support default|cadvisor|conformance")
+var _ = flag.String("system-spec-name", "", fmt.Sprintf("The name of the system spec used for validating the image in the node conformance test. The specs are at %s. If unspecified, the default built-in spec (system.DefaultSpec) will be used.", system.SystemSpecPath))
+var _ = flag.String("extra-envs", "", "The extra environment variables needed for node e2e tests. Format: a list of key=value pairs, e.g., env1=val1,env2=val2")
+var _ = flag.String("runtime-config", "", "The runtime configuration for the API server on the node e2e tests.. Format: a list of key=value pairs, e.g., env1=val1,env2=val2")
+var _ = flag.String("kubelet-config-file", "", "The KubeletConfiguration file that should be applied to the kubelet")
 
 func main() {
 	klog.InitFlags(nil)

